### PR TITLE
Axe 897

### PR DESCRIPTION
**Pull request by jlotery**
Global test script renamed to "test" in package.json.
All tests refactored so that all tests can be run with a single mocha command. 
Running all tests produces same results as running single tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,19 +6,15 @@
...
-    "test-all": "npm run test-assets; npm run test-auth; npm run test-docs; npm run test-errors; npm run test-modules; npm run test-prefix; npm run test-services; npm run test-trace; npm run test-webhooks; npm run test-versioned-docs"
+    "test-campsi": "mocha -R dot \"./test/campsi/*.js\" --exit",
+    "test-assets": "mocha -R dot \"./test/assets/*.js\" --exit",
+    "test-auth": "mocha -R dot \"./test/auth/*.js\" --exit",
+    "test-docs": "mocha -R dot \"./test/docs/*.js\" --exit",
+    "test-webhooks": "mocha -R dot \"./test/webhooks/*.js\" --exit",
+    "test-trace": "mocha -R dot \"./test/trace/*.js\" --exit",
+    "test-versioned-docs" :"mocha -R dot \"./test/versioned-docs/*.js\" --exit",
+    "test-stripe-billing" : "mocha -R dot \"./test/stripe-billing/*.js\" --exit",
+    "test": " mocha './test/**/*.js' --exit"
```
Comment (ChristopheBraud) [inline package.json:17]: How do you ignore config folders ? You showed me a version that ignore those folders if I remember well ?

**Review by ChristopheBraud — CHANGES_REQUESTED** [review]
It would have been better to move index files you deleted in a special "launch test config" folder. The purpose is to test our code with Postman during development, or to help preparing tests' config. The goal in the PR is to move those files from test folder to simplify mochas command and have a better tree architecture

**Review by ChristopheBraud — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (jlotery) [reply package.json:17]: I realized I didn't need to do that as there are no test entry points in the folders that were being ignored. If you think it's better to exclude the directories that do not contain tests let me know and I'll revert back to that.

**Pull request merged**